core/templates/dev/head/components/SearchBarDirective.js

```diff
@@ -21,13 +21,13 @@ oppia.directive('searchBar', [function() {
     restrict: 'E',
     templateUrl: 'components/searchBar',
     controller: [
-      '$scope', '$rootScope', '$timeout', '$window', 'searchService',
-      'oppiaDebouncer', 'ExplorationCreationButtonService', 'urlService',
-      'CATEGORY_LIST',
+      '$scope', '$rootScope', '$timeout', '$window', '$location',
+      'searchService', 'oppiaDebouncer', 'oppiaHtmlEscaper',
+      'ExplorationCreationButtonService', 'urlService', 'CATEGORY_LIST',
       function(
-          $scope, $rootScope, $timeout, $window, searchService,
-          oppiaDebouncer, ExplorationCreationButtonService, urlService,
-          CATEGORY_LIST) {
+          $scope, $rootScope, $timeout, $window, $location, searchService,
```

Comment:
A really minor nit: we generally try to sync the line breaks here with those in line 24 - 26, to make it easy to compare the dependency lists and ensure they're the same. So, break after $location, oppiaHtmlEscaper, etc.